nemo/collections/nlp/modules/common/bert_module.py

```diff
@@ -79,8 +79,12 @@ def input_example(self, max_batch=1, max_dim=256):
         """
         sample = next(self.parameters())
         sz = (max_batch, max_dim)
-        input_ids = torch.randint(low=0, high=2048, size=sz, device=sample.device)
+        input_ids = torch.randint(low=0, high=max_dim - 1, size=sz, device=sample.device)
```

Comment:
Any reason for clamping high value to max_dim-1?
Shouldn't  `high` value depend on the size of the vocabulary?